### ISSUE TITLE
`process.env` is undefined and causes application-crashing error in non-node runtime

### ISSUE THREAD

**Issue by AsukaCHikaru** [Anber/wyw-in-js#92]
## Environment

- wyw-in-js version: 0.5.3 (@linaria/core@6.2.0)
- Bundler (+ version): vite@5.3.3
- Node.js version: Deno 1.44.4
- OS: MacOS 13.4, Apple silicon

## Description

In a non-node environment, `process.env` is undefined thus causing `packages/shared/src/findPackageJSON.ts` L13 to throw an undefined error which leads to the application crashing.

```ts
import { dirname, isAbsolute } from 'path';
import * as process from 'process';

import findUp from 'find-up';

const cache = new Map<string, string | undefined>();

export function findPackageJSON(
  pkgName: string,
  filename: string | null | undefined
) {
  // Jest's resolver does not work properly with `moduleNameMapper` when `paths` are defined
  const isJest = Boolean(process.env.JEST_WORKER_ID);
  ...
```

## Reproducible Demo

https://github.com/AsukaCHikaru/wyw-in-js-vite-env-issue

**Comment by AsukaCHikaru** [comment]
solved in #93

**Comment by AsukaCHikaru** [comment]
solved in #93
